package.json

```diff
@@ -23,13 +23,21 @@
   },
   "devDependencies": {
     "@commitlint/config-conventional": "^17.0.2",
+    "@eslint/compat": "^1.2.7",
+    "@eslint/eslintrc": "^3.3.0",
+    "@eslint/js": "^9.22.0",
     "@evilmartians/lefthook": "^1.5.0",
     "@lerna-lite/cli": "^1.13.0",
     "@lerna-lite/run": "^1.13.0",
     "commitlint": "^17.0.2",
     "concurrently": "^7.2.2",
-    "eslint": "^8.52.0",
+    "eslint": "^9.22.0",
     "eslint-config-satya164": "^3.2.0",
+    "eslint-plugin-react-dom": "^1.35.0",
+    "eslint-plugin-react-hooks-extra": "^1.35.0",
+    "eslint-plugin-react-naming-convention": "^1.35.0",
+    "eslint-plugin-react-web-api": "^1.35.0",
+    "globals": "^16.0.0",
```

Comment:
`@eslint/compat`, `@eslint/eslintrc`, `@eslint/js` and `globals` are required for the new Flat Config format. The other dependencies are required to make it all work with [satya164](https://github.com/satya164/eslint-config-satya164)'s config